training/run_benchmarks/run.py

```diff
@@ -322,16 +367,46 @@ def start_tasks_in_cluster(dp_path, container_name, case_config, base_args,
     time.sleep(60)
 
 
-def wait_for_finish(dp_path, container_name, pid_file_path, nnodes):
+def is_pid_running(pid_file_path):
+    '''Return whether the process with pid is running in container.
+        Return value:
+            True: It is running.
+            False: It isn't running.
+    '''
+    get_pid_cmd = "cat " + pid_file_path
+    ret, outs = run_cmd.run_cmd_wait(get_pid_cmd, 5)
+    if ret == 0:
+        task_pid = int(outs[0])
+    else:
+        print("Can't find pid file ", pid_file_path, "in container.")
+        return False
+    check_cmd = "ls /proc/" + str(task_pid) + "/cmdline"
+    ret, outs = run_cmd.run_cmd_wait(check_cmd, detach=False)
+    if ret == 0:
+        print("The process is running.")
+        return True
+    print("The process is not running.")
+    return False
+
+
+def wait_for_finish(dp_path, container_name, pid_file_path, nnodes, env_flag):
     '''wait all the processes of start_xxx_task.py finished.
     '''
+    # if env_flag == "baremetal":
     # Aussme pid of start_xxx_task.py won't loop in a short time.
     check_cmd = "cd " + dp_path + "; " + sys.executable \
                 + " utils/container_manager.py -o pidrunning -c " \
                 + container_name + " -f " + pid_file_path
-
+    # if env_flag == "container":
+    #     # TODO 如果原本就在容器内，如何判断？
```

Comment:
if start task in container, there's no need to "wait finishi", because you don't need to stop the container